packages/api-client/src/index.ts

```diff
@@ -1,3 +1,5 @@
 export * from './api';
 export * from './data';
 export * from './models';
+
+export { default as ApiClientFactory } from './api/ApiClientFactory';
```

Comment:
^ this is wrong.  It works, but typically the new class would be imported and reexported in `/src/api/index.ts`, and this file would be untouched.  I just couldn't get it to work the way I have on other projects and I was wasting a lot of time on it.  If this is good enough, that's fine, it works, else I would greatly appreciate advice on what I'm doing wrong.

Reply:
fixed